docker_images/sklearn/tests/generators/tabularregression.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+# This is an amost 1:1 copy of iris.py but uses regression instead of
```

Comment:
should we then just put all of this in one single file?

Reply:
Do you mean to put it into a single script as is, or try to make it more generic so that some functions can be re-used?

Reply:
I think with a bit of tweak the code in these two files can be made a bit more generic to avoid duplication.

Reply:
Done, please take a look at `generate.py`